.github/workflows/endToEndTests.yml

```diff
@@ -0,0 +1,40 @@
+name: Test end to end
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - release
+  push:
+    branches:
+      - develop
+      - release
```

Comment:
this is one thing I'm not sure about — when should the end-to-end tests run? Right now I have it on any PR to `develop` or `release`, should I also be having it run once they merge into `develop` and `release` as well?

Reply:
Seems fine for now, I'll be working on the e2e suite so I'll revisit once I look at it a bit more